packages/grafana-ui/src/components/EmptyState/EmptySearchState/EmptySearchState.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactNode } from 'react';
+
+import { t } from '../../../utils/i18n';
+import { Box } from '../../Layout/Box/Box';
+import { Stack } from '../../Layout/Stack/Stack';
+import { Text } from '../../Text/Text';
+
+import { GrotNotFound } from './GrotNotFound/GrotNotFound';
+
+interface Props {
+  image?: ReactNode;
+  message?: string;
+  showImage?: boolean;
+}
+
+export const EmptySearchState = ({
+  children,
+  image = <GrotNotFound width={300} />,
```

Comment:
I think that this should default to _not_ showing the grot image. I think the image should be used sparingly so it's more an occasional surprise/delight, rather than a full-frontal assualt of grots everywhere.

Reply:
my 2c: i don't think empty search states are a full-frontal assault 😛 imo they're a very common place for an image. without it, and without supporting text (which would be a lot of the use cases) this component is **very** boring 😅 

![image](https://github.com/grafana/grafana/assets/20999846/6cd54bf5-2125-44aa-a75d-c13bed18c466)